src/languageserverinstance.jl

```diff
@@ -40,6 +40,7 @@ mutable struct LanguageServerInstance
     symbol_store::Dict{Symbol,SymbolServer.ModuleStore}
     symbol_extends::Dict{SymbolServer.VarRef,Vector{SymbolServer.VarRef}}
     symbol_store_ready::Bool
+    workspacepackages::Dict{String,Document}
```

Comment:
This should probably have UUIDs as the key, right? I think we are generally not identifying packages by their UUID right now, but probably should to get this right.